### DISCUSSION TITLE
Craft5 grouping fields and entry types in the CP

### DISCUSSION THREAD

**Discussion by FrDH** [craftcms/cms#15546] [Ideas]
IMO we need to be able to group fields and entry types in the CP.
Currently, if I go to `cp/settings/entry-types` I get a list all entry types, those for entries and those for matrix fields all mixed together. It is very confusing.
Same goes for `cp/settings/fields`.

**Comment by alexanderbuergin** [comment]
@brandonkelly: We often have the same problem. Readding the option to add groups to the fields and entry types would be very useful.

**Reply by wvgogh** [reply]
I'd love to group entry types as well. Now we just get a big list of blocks and it only gets bigger and bigger. Would love to see a native way to group the element so the UX for the content manager would improve a lot.

![image](https://github.com/user-attachments/assets/65cc5c93-6c59-469b-9bb2-f5f1891aec72)

**Comment by FrDH** [comment]
For now, I resorted to "grouping" entry types by adding colors and prefixing the names. Both are far from ideal because they're visible to the authors.

**Reply by janhenckens** [reply]
We're also using/abusing the color options for this (Blue for everything that's in a Matrix, another color for everything else for now)

**Comment by daltonrooney** [comment]
This is becoming an issue for us as well. We have fewer fields in Craft 5 than we did in Craft 4, but grouping entry types would be a big improvement! It would be great if we could group entry types in the CP and even better if we could group them within the matrix field "add" button dropdown.

**Reply by alexanderbuergin** [reply]
Yes. That would boost UX massively

**Comment by FrDH** [comment]
Just another thought on how to simulate grouping entry types: Add a prefix to the handle and order them by handle. 
At least this way the authors won't be bothered with apparently random icons, prefixes or colors.

**Reply by alexanderbuergin** [reply]
Currently, we do it like this
